libcudacxx/include/cuda/std/detail/libcxx/include/complex

```diff
@@ -433,6 +433,18 @@ inline constexpr bool __is_complex_v = false;
 template <class _Tp>
 inline constexpr bool __is_complex_v<complex<_Tp>> = true;
 
+template <typename _Tp>
+inline constexpr int __num_bits_v<complex<_Tp>> = __num_bits_impl<remove_cv_t<_Tp>>() * 2;
+
+template <typename _Tp>
+inline constexpr int __num_bits_v<const complex<_Tp>> = __num_bits_impl<remove_cv_t<_Tp>>() * 2;
+
+template <typename _Tp>
+inline constexpr int __num_bits_v<volatile complex<_Tp>> = __num_bits_impl<remove_cv_t<_Tp>>() * 2;
+
+template <typename _Tp>
+inline constexpr int __num_bits_v<const volatile complex<_Tp>> = __num_bits_impl<remove_cv_t<_Tp>>() * 2;
+
```

Comment:
Only the first one should be necessary, if we would specialize `__num_bits_impl` rather than `__num_bits_v`